apidoc/Titanium/UI/Slider.yml

```diff
@@ -126,6 +126,7 @@ events:
         summary: |
             Dictionary with properties `width` and `height` of the size of the thumb.
             Available with custom thumb image.
+        type: Dictionary
```

Comment:
Should probably define a simple type for this. It's just an Object with `width` and `height` properties, which are both `Number` type.

Reply:
Note that the most relevant is our `Dimension` type declared in Ti.Ui.View apidocs.

That type has width, height, x, and y; and it is used for the `rect` and `size` properties for `View`s (which can be access after `postlayout` is fired)

In fact, we basically even say the `size` property's `Dimension` just has `x` and `y` === 0; which says to me we should really have a `Size` type which holds `width` and `height`, that `Ti.Ui.View.size` should return that, and that `Dimension` should be a subclass that adds `x` and `y` (and is returned by `Ti.Ui.View.rect`)

(Also, Android has optional properties on Dimension of `absoluteX` and `absoluteY`, which aren't documented on that type, but in `Ti.Ui.View.rect`!)

Reply:
Don't think there is a way to describe properties of a property. The only solution I know is to define additional "pseudo" type like `Dimension` or `AnimationOption`. (btw, docs have ~230 other properties typed with `type: Dictionary`).

But I definitely want a way to describe them all.

Reply:
Yeah that's what I was getting at. Ideally we'd define the "pseudo" type `Size` with just `width` and `height`, modify the `Dimension` "pseudo" type to extend it.

Also, good point that `Dictionary` has been used as a catch-all. We should warn (not error) about it, since really it's another way of saying "generic `Object`" and over time should be cleaned up/defined.